ivy/functional/backends/numpy/activations.py

```diff
@@ -115,3 +115,7 @@ def mish(x: np.ndarray, /, *, out: Optional[np.ndarray] = None) -> np.ndarray:
 
 
 mish.support_native_out = True
+
+
+def logsigmoid(input: np.ndarray) -> np.ndarray:
+    return -(np.log1p(np.exp(-(input))))
```

Comment:
I think you're doing this wrong mathematically. 
It should be log of the inverse of 1+exp, not log of 1+exp. Have a look at how pytorch defines this:
![image](https://user-images.githubusercontent.com/111577222/225381529-ac4caef6-b101-4274-a3ea-2b6d458d9559.png).
This might be the cause of the failure.

Reply:
Hey. Actually these two are mathematically equivalent due to the properties of logarithms. For example, take a look how Jax defines it:

![image](https://user-images.githubusercontent.com/70361308/225394168-aa2505f3-5531-45fd-83dc-5138f70ab6fd.png)

Reply:
*facepalm* yeah I forgot about that